CONTRIBUTING.md

```diff
@@ -229,16 +230,28 @@ $ genhtml -o html_report coverage.info
 
 ## Debugging
 
-### Checking the UMF version and CMake variables (Linux only)
+### Checking UMF version and build options
 
-Strings with the UMF version and useful CMake variables can be grepped in the following way:
+From an already built UMF shared library you can obtain UMF precise version and
+CMake variables/options it was built with. It's not only useful to verify what should
+be included within the library, but also for debugging. If you're filing an issue to
+UMF project, please include this information in your ticket.
+
+#### Linux
+
+It requires `binutils` package installed in the system. Then, it's best to grep it
```

Comment:
I don't like this fragment ... please consider changing it to something like "Make sure the binutils package is installed on your system. Then, you can use the following grep command:"

Reply:
Done.